Makefile

```diff
@@ -126,15 +126,13 @@ github_release: update_docs
 
 pypi_release: # Has an implicit dependency on github_release because...
 	# Upload to PyPI, automatically picking tag created by github_release
-	cd release && python3 -m venv venv
-	cd release && venv/bin/pip3 install --upgrade setuptools pip
-	cd release && venv/bin/pip3 install twine wheel
-	cd release && venv/bin/python3 setup.py sdist bdist_wheel
+	python3 -m venv venv
+	venv/bin/pip3 install --upgrade pip build twine
+	venv/bin/pyproject-build
```

Comment:
[`build`](https://pypi.org/project/build/) is simple front end for building Python packages that implement PEP 517. This means that should you ever move from setuptools to a different build backend, this line doesn't have to change and will still work.